### DISCUSSION TITLE
Is there a Box concept?

### DISCUSSION THREAD

**Discussion by sturlath** [Megabit/Blazorise#2484] [Q&A] [answered]
Now I need to create a square element with coloured background and centred text etc. that basically looks like a button. I would use the button element if I could disable the hover over effects.

And that got me thinking about if we could have some kind of [box model](https://www.w3schools.com/css/css_boxmodel.asp) element in Blazorise?

> "The box model allows us to add a border around elements, and to define space between elements. "

**Comment by stsrki** [comment, marked as answer]
This is easy in v0.9.4 with new utilities

![image](https://user-images.githubusercontent.com/900302/122289347-e687d080-cef2-11eb-8c76-a851a684c0bd.png)

```cshtml
<Div Border="Border.Is1.Success" Shadow="Shadow.Default" Padding="Padding.Is3" Margin="Margin.Is2" Style="width: 300px;">
    This text is the content of the box. We have added a size 3 padding, size 2 margin and a green border. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
</Div>
```

**Reply by sturlath** [reply]
It seams that ALL my problems will be fixed with that version 😄 I think Ill just put of any UI detail until then and focus on  business logic 😉 you will not get one more question until then, I promise!
